Lib/test/test_asyncio/test_subprocess.py

```diff
@@ -35,6 +35,17 @@
               'data = sys.stdin.buffer.read()',
               'sys.stdout.buffer.write(data)'))]
 
+# Program generating infinite data
+PROGRAM_YES = [
```

Comment:
Is there a reason to make this a global var? Maybe we can just define it in test?

Reply:
No reason other than consistency with `PROGRAM_CAT` and `PROGRAM_BLOCKED`. I can make it a local variable.